.vscode/launch.json

```diff
@@ -226,6 +226,52 @@
             },
             "cwd": "${workspaceRoot}/magda-gateway"
         },
+        {
+            "type": "node",
+            "request": "launch",
+            "protocol": "inspector",
+            "name": "Launch Gateway for Terria",
```

Comment:
This Terria-specific stuff doesn't belong here.

Reply:
Removed.